src/python/libcgroup.pyx

```diff
@@ -310,6 +318,43 @@ cdef class Cgroup:
             i = i + 1
         return mount_points
 
+    @staticmethod
+    def create_scope(scope_name, pid=None, slice_name='libcgroup.slice', delegated=True,
```

Comment:
should we follow the C definition and have `def create_scope(scope_name, slice_name, delegated, systemd_mode, pid)`
and have default scope name too `scope_name='libcgroup.scope`.  As a side effect, the user can create a default slice and scope with no arguments passed.  In the C version, the   `systemd_mode` is initialized to `CGROUP_SYSTEMD_MODE_FAIL`

Reply:
Yeah that sounds like a good plan to me

Reply:
Done